### ISSUE TITLE
Test the gcc compiler for CVA6 and llvm-generic for snitch core in SNAX Container

### ISSUE THREAD

**Issue by IveanEx** [KULeuven-MICAS/HeMAiA#18]
We are going to abandon the original Occamy docker image and use SNAX image for both hardware and software generation. The toolchain is already available in [KULeuven-MICAS/snax_cluster#234](https://github.com/KULeuven-MICAS/snax_cluster/pull/234) and we need to do several modifications to use it: 

- [ ] Bootrom: modify the Makefile in ***target/rtl/bootrom*** and ***target/fpga/bootrom*** and validate the correct compilation by downloading the Bootrom on FPGA
- [ ] Device Software: normally nothing needs to be modified. Just to validate if the software is linked at the correct memory offset
- [ ] Host Software: modify the toolchain and test the compilation. Make sure that the correct loader script is emitted during the first compilation; and the final linking is conducted correctly.

**Comment by IveanEx** [comment]
Please wait [KULeuven-MICAS/snax_cluster#234](https://github.com/KULeuven-MICAS/snax_cluster/pull/234) to be merged into main before starting the actual work.
